### PR TITLE
Feature/network selection

### PR DESCRIPTION
**Pull request by LukeHackett12**
Hey! Hoped I'd get you're thoughts on the below changes. 

I didn't see a way to select a network the way it specifies in you're api docs [here](https://api.rated.network/docs), so I added an optional config for it. I also updated the readme with an example but indicated it was not necessary. Let me know what you think, thanks!

Also added a public licence if that works for you guys.

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 rated-network
```
Comment (ariskk) [inline LICENCE]: Copyright (c) 2022 Rated Labs

File: LICENCE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 rated-network
+Copyright (c) 2022 Rated Labs
```
Comment (ariskk) [inline LICENCE:3]: You can squash/fixup this commit in the previous one.

**Review by ariskk — APPROVED** [review]
Just squash the last two commits, then LGTM. Thanks!

**Comment by LukeHackett12** [comment]
Thanks for the quick review!

**Pull request merged**